src/app/components/UserAssets/index.module.scss

```diff
@@ -0,0 +1,5 @@
+.actionLink {
+  @apply tw-text-primary tw-tracking-normal hover:tw-text-primary hover:tw-underline tw-mr-1 xl:tw-mr-4 tw-p-0 tw-font-normal;
+
+  font-family: Montserrat;
```

Comment:
There is no need to specify `font-family` like this because we already have `Montserat` defined in `tailwind.config.ts` so you could just add `tw-font-body` to `@apply`.

Since you have no custom property here, you could delete this selector and the whole file (but I can see it's used on 4 places so I'll leave it up to you).